libs/workorderapp/src/lib/config/tableConfig.tsx

```diff
@@ -196,14 +196,34 @@ const columnDefinitions: [ColDef<WorkOrder>, ...ColDef<WorkOrder>[]] = [
   },
   {
     colId: 'WorkBreakdownStructure',
-    headerName: domainNames.wbs,
-    headerTooltip: domainNames.wbs,
+    headerName: domainNames.workorderWBS,
+    headerTooltip: domainNames.workorderWBS,
     valueGetter: (pkg) => pkg.data?.workBreakdownStructure,
   },
   {
     colId: 'Systems',
-    headerName: domainNames.systems,
-    headerTooltip: domainNames.systems,
+    headerName: domainNames.commSystems,
+    headerTooltip: domainNames.commSystems,
     valueGetter: (pkg) => pkg.data?.systems,
   },
+  {
+    colId: 'EstimatedHours',
+    headerName: domainNames.remainingManHours,
+    headerTooltip: domainNames.remainingManHours,
+    valueGetter: (pkg) => pkg.data?.remainingHours,
+    cellRenderer: (props: ICellRendererProps<WorkOrder, string>) => {
+      if (!props.value) return null;
+      return <StyledMonospace>{props.value ?? null}</StyledMonospace>;
+    },
+  },
+  {
+    colId: 'RemainingHours',
+    headerName: domainNames.estimatedManHours,
+    headerTooltip: domainNames.estimatedManHours,
+    valueGetter: (pkg) => pkg.data?.estimatedHours,
+    cellRenderer: (props: ICellRendererProps<WorkOrder, string>) => {

```

Comment:
Why are we having a guard clause and handeling the null value later on?
Can we remove the guard clause?